Code/GraphMol/ChemTransforms/MolFragmenter.h

```diff
@@ -122,7 +122,14 @@ struct RDKIT_CHEMTRANSFORMS_EXPORT MolzipParams {
   std::vector<std::string> atomSymbols;
   std::string atomProperty;
   bool enforceValenceRules = true;
-  bool generateCoordinates = false;
+  bool generateCoordinates =
+      false;  //< if true, 2D coords will be generated for
+              //  the output molecule and assigned to the input fragments as
```

Comment:
Is the //< a typo, or some extra comment markup for the docs that I don't recognise?

Reply:
It's a doxygen thing, but I think I mis-spelled it: should probably be: ///<